### DISCUSSION TITLE
Malfunction in m.naver.com on website's darkmode

### DISCUSSION THREAD

**Discussion by I0rem1psum** [List-KR/community#38] [General]
I've recently found my adguard premium for ios and list-kr filter does not work in m.naver.com properly, showing its 'Powerlink Ads'.

In my case, I solved this problem switching from website's built-in darkmode(as known as System default(기기설정)) to Lightmode(라이트모드).

I've just found that list-kr is properly blocking 'Powerlink Ads', when m.naver.com is on its lightmode.

But I still don't know the way it does.

Does anyone already know this problem?

**Comment by seia-soto** [comment]
Hello, I think I am the only person who can test on iOS. Can you tell me the exact settings? By assuming your setting, you're likely saying the ad blocking on `m.naver.com` with darkmode enabled did not work properly. Did I understand correctly?

**Reply by I0rem1psum** [reply]
hello, yes, exactly.

**Reply by seia-soto** [reply]
Thanks! Also, for the faster processing, can you upload a screenshot?

**Reply by I0rem1psum** [reply]
<details>
<summary>Details</summary>

![dark_menu2](https://user-images.githubusercontent.com/116969691/199871023-a54da449-2694-434f-ab7d-5864fe6e35a9.png)
![dark_result](https://user-images.githubusercontent.com/116969691/199871032-f261d24e-05b7-4c6d-bc9a-bcd8226135bb.png)
![Light_main](https://user-images.githubusercontent.com/116969691/199871039-6b242a8b-3878-4073-924e-bc773c5efd77.png)
![Light_menu1](https://user-images.githubusercontent.com/116969691/199871042-b6b2f44d-9898-4770-9d98-d0c9c7aa91c1.png)
![Light_menu2](https://user-images.githubusercontent.com/116969691/199871045-8f9b52ee-a6e7-4cf3-913a-bfb4d8734a68.png)
![Light_result](https://user-images.githubusercontent.com/116969691/199871047-8e2db4b1-23cf-40cd-a7c0-27227f0d8548.png)

</details>

**Reply by I0rem1psum** [reply]
please access the option clicking circled buttons.

**Reply by seia-soto** [reply]
Thanks for waiting. I did some test but couldn't reproduce your case, using List-KR version 2022.1103.3 on AdGuard Pro for iOS. Please, can you try updating the filter on the latest version of AdGuard?

<details>
<summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/30369714/199872319-ccfd968b-9f4a-459f-893e-05e6f7399207.png)

</details>

**Reply by I0rem1psum** [reply]
I've updated Adguard app and Adguard filters already and it's same version with you, but nothing changed. the only thing that differs from you is that I'm using Adguard app with Premium Subscription, and you are using AdGuard Pro for iOS.

**Reply by seia-soto** [reply]
Hi! Thanks for noticing me. I setup the bare-AdgGuard for Safari extension on my mac and could reproduce your case. Can you test if `m.search.naver.com##.sc._pwl_video_container` working? You can add it into User Rules section under Safari Content Blocking settings.

<details>
<summary>Details</summary>

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/30369714/199873617-45731565-2553-4af2-b301-6d6b62e1ce6d.png">

</details>

**Reply by I0rem1psum** [reply]
Thank you ! it's working with your user rules!

<details>
<summary>Details</summary>

![IMG_3059](https://user-images.githubusercontent.com/116969691/199874000-7c0e563f-fe6a-40cd-ae63-70b63234fbd4.PNG)

</details>

**Comment by seia-soto** [comment]
Completing this via #562 .
